stats/worker_commands.go

```diff
@@ -102,15 +96,12 @@ type getViewByNameResp struct {
 
 func (cmd *getViewByNameReq) handleCommand(w *worker) {
 	if v, ok := w.viewsByName[cmd.name]; ok {
-		cmd.c <- &getViewByNameResp{
-			v,
-			nil,
-		}
+		cmd.c <- &getViewByNameResp{v, nil}
```

Comment:
Can we enforce golang format for the repository? Looks like a lot of this kind of changes. Maybe file an issue.

Reply:
The packages are already gofmt'ed and we have a CI rule to enforce it before merge. gofmt'ed is not opinionated on everything. Multiple lines are permitted but hard to read.